.travis.yml

```diff
@@ -0,0 +1,20 @@
+#language: ruby
+
+sudo: true
+
+install:
+  # init/start docker in travis
+  - source <(curl -sL https://raw.githubusercontent.com/zuazo/kitchen-in-travis/master/scripts/start_docker.sh)
+  - DOCKER_IMAGE=${DOCKER_IMAGE:-"ubuntu:16.04"}
+  - DOCKER_OPTS="${DOCKER_OPTS} -e DEBIAN_FRONTEND=noninteractive -h cfg01.ci.local"
+
+script:
+  - cd $TRAVIS_BUILD_DIR
+  - ID=$(docker run ${DOCKER_OPTS} -v $(pwd):/tmp/reclass -i -t -d ${DOCKER_IMAGE})
```

Comment:
not needed for the script/jenkins. It droped from a config where I used kitchen.ci to run the jobs (it mount's the local directory and copy file (so local's are not modified any way). The script was basically shared.